src/web/event-notifications/event-notification-types/SlackNotificationForm.jsx

```diff
@@ -144,8 +148,8 @@ class SlackNotificationForm extends React.Component {
     render() {
       const { config, validation } = this.props;
       const { isBacklogSizeEnabled, backlogSize } = this.state;
-      const url = 'https://docs.graylog.org/en/latest/pages/alerts.html#data-available-to-notifications';
-      const element = <p>Custom message to be appended below the alert title. See <a href={url} rel="noopener noreferrer" target="_blank">docs </a>for more details.</p>;
+      const url = 'https://docs.graylog.org/docs/alerts#notifications';
```

Comment:
Passing this URL to `DocumentationLink` will duplicate the origin and `/docs` path, resulting in `https://docs.graylog.org/docs/https://docs.graylog.org/docs/alerts#notifications`.